squads/domani-ai/config/source-tree.md

```
@@ -0,0 +1,14 @@
+# Source Tree — domani-ai Squad
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)